src/views/RecIM/views/Activity/index.jsx

```diff
@@ -241,17 +241,6 @@ const Activity = () => {
                 </Grid>
               </Grid>
             </Grid>
-            {openTeamForm && (
-              <TeamForm
-                closeWithSnackbar={(teamID, status) => {
-                  handleTeamFormSubmit(status, setOpenTeamForm);
-                  navigate(`${activityID}/team/${teamID}`);
-                }}
-                openTeamForm={openTeamForm}
-                setOpenTeamForm={(bool) => setOpenTeamForm(bool)}
-                activityID={activityID}
-              />
-            )}
```

Comment:
duplicated from auto-merge

Reply:
Yup! Remember seeing this from when I was working with you.